content/src/ports/activeEntities.ts

```diff
@@ -106,7 +106,7 @@ export const createActiveEntitiesComponent = (
   const update = async (pointers: string[], entity: Entity | NotActiveEntity): Promise<void> => {
     for (const pointer of pointers) {
       setPreviousEntityAsNone(pointer)
-      entityIdByPointers.set(pointer, isEntityPresent(entity) ? entity.id : entity)
+      entityIdByPointers.set(normalizePointer(pointer), isEntityPresent(entity) ? entity.id : entity)
```

Comment:
should we do something similar for the queries by entityId and URN prefix?

Reply:
I'm on it, thanks! Will test the id workflow as well.

Reply:
I updated the approach to also cover the edge cases related to the ids. Anyways I still have doubts about the entities id management in the database, if the database is case insensitive by always using lowercases this solution is safe, otherwise I think this won't work as expected and we will have further bugs.

Reply:
These changes about normalizing also the ids fixed the corner case explained in this PR description -> https://github.com/decentraland/catalyst/pull/1380

Reply:
After confirming that the entity ids should be case sensitive we reverted this change. Anyways, we took advantage of these modifications to solve the corner case explained in the PR description.